core/controllers/access_validators_test.py

```
@@ -668,6 +668,174 @@ def test_any_user_can_access_end_of_arc_page(self) -> None:
+            ],
+            'Added acquired skill IDs and arc.',
+        )
+
+        self.get_html_response(
+            '%s/can_access_practice_session_page/%s/%s/test/arc/%s'
+            % (
+                ACCESS_VALIDATION_HANDLER_PREFIX,
+                'math',
+                'public-topic-name',
+                'default_arc',
+            ),
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)